user/container/Makefile

```diff
@@ -57,18 +58,21 @@ run:
 	$(info Arguments: NAME=${NAME})
 	$(info Arguments: BASEPATH=${BASEPATH})
 	$(info Arguments: EXTERNAL_HOSTNAME=${EXTERNAL_HOSTNAME})
-	mkdir -p ${BASEPATH}/.kcp ${BASEPATH}/kubestellar-logs
+	$(info Arguments: EXTERNAL_PORT=${EXTERNAL_PORT})
 	docker run -d \
-		-p 6443:6443 \
-		-v ${BASEPATH}/.kcp:/.kcp \
-		-v ${BASEPATH}/kubestellar-logs:/kubestellar-logs \
```

Comment:
With these (thankfully) gone, I suspect there is no need for the `--user` flag below (which is countermanding what is in the Dockerfile anyway).

Reply:
ok, you are right